src/core/lifetime.d

```diff
@@ -2154,7 +2154,7 @@ private void moveEmplaceImpl(T)(scope ref T target, return scope ref T source)
     else static if (__traits(isStaticArray, T))
     {
         for (size_t i = 0; i < source.length; ++i)
```

Comment:
Shouldn't this have a special casing that resorts to a `memcpy` when `target` is a type lacking copy-constructor nor postblit?